src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,60 @@
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+  slugSelected?: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slugSelected }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+      <tbody>
+        {people.map(personList => {
```

Comment:
```suggestion
        {people.map(person => {
```
The name is a bit confusing as it's just an object of person